---
layout: default
---

* Principal investigator: Anna Reymer ([contact info at Gothenburg University](https://www.gu.se/en/about/find-staff/annareymer)).

We are a computational chemistry team working to decipher the regulatory logic of eukaryotic transcriptional control, to create the basis for new gene-editing therapies. We seek to tackle some of the trickiest aspects of eukaryotic transcriptional control, which due to their dynamic nature often extremely difficult to address by direct experimental observation but can be approached theoretically. Our tools include classical computational chemistry techniques, such as MD and QM, molecular structural design, but also bioinformatics data mining and machine learning. 

Would you like to join us? We are always looking for talented Postdocs, PhD students, MSc and BSc students. If you are interested in our work, please contact [Anna Reymer](mailto:anna.reymer@gu.se).

### Current projects:
* Role of DNA sequence-specific flexibility for selective transcription factor proteins-DNA binding
*	Molecular mechanisms of collaborative transcription factor proteins-DNA recognition
*	Impact of single nucleotide polymorphisms and DNA methylation on transcriptional control
*	Transcription factors interactions with long non-coding RNA
*	Alternative RNA splicing
*	Design of therapeutic nucleic acids

### Publications
Papers: see [Google Scholar](https://scholar.google.com/citations?user=4TfCOOIAAAAJ).

Open-source software:
* [DNA Deformation Toolkit](https://github.com/annareym/DNA_Deformation_Toolkit).
* [DNA Twisting tool](https://github.com/annareym/PLUMED_DNA-Twist).

### Funding

* [Swedish Foundation for Strategic Research SSF](https://strategiska.se/pressmeddelande/236-miljoner-till-instrument-teknik-och-metodutveckling/) 
* [Hasselblad Foundation Prize for outstanding female researchers](https://www.mynewsdesk.com/se/hasselblad-foundation/pressreleases/kvinnliga-forskare-faar-miljonanslag-2306012) 
* Swedish Research Council VR


### Popular scientific description of team's work

DNA - the molecule of life - carries coded instructions on every aspect of any organism’s live: how to make different building blocks of cells, such as proteins and RNA, as well as instructions that regulate all biological processes in cells. DNA sequence coding and non-coding regions. The coding regions contain the information about proteins and RNA. The non-coding regions, which for a very long time were believed to just be the genetic junk yard of evolution, appear to play a regulatory role! For example, non-coding DNA contain special regions, operons, which are attractive to certain regulatory proteins. When proteins bind, they deform DNA in a specific way and it is believed that the deformations (the "softness" of DNA) help the proteins recognize their proper binding sites and then initiate biological reactions, such as DNA transcription. 

DNA transcription is a process during which a portion of a DNA sequence is copied to an RNA molecule that can have a role on its own or can be used for the synthesis of protein molecules. The number of RNA molecules, of any sort, produced during one transcription event must correspond exactly to the cell's needs - this is what we call the correct regulation of DNA transcription. If the number is too big or too little various pathogenic events may arise, firstly on a cellular level, that gradually might create a problem for the organism as a whole, such as cancers, autoimmune diseases, Alzheimer, etc. 

Despite many decades of intensive research, we still do not completely understand how the transcription reaction starts and how each cell "determines" the number of copies of different RNA molecules it needs at a given time to stop the transcription reaction. In my team we aim to provide answers on what happens in the early moments of DNA transcription reaction and describe these molecular processes with atomistic resolution. 

We know that the beginning of the DNA transcription reaction in eukaryotes – all organisms from yeast to human ¬– are controlled by so-called transcription factor proteins. There are different types of transcription factor proteins that can bind in different fashion to diverse stretches of non-coding DNA to realize their transcriptional regulatory programs. Transcription factor proteins can work alone or in combination with each other, which allows to increase the diversity of non-coding DNA recognized by these combinations of proteins. 

The extremely dynamic and complex nature of the DNA transcription reaction makes the details of the biological process difficult to observe experimentally. In this situation, the so-called computational methods, which takes the help of computers to create digital models and simulates the various steps in the process, contributes to research progress. Development of new computational methods and increased computational power have made molecular modeling a kind of "in-silico microscope" that we can use to test different hypotheses and scenarios of how biological reactions can take place.

In our research, we use a combination of interdisciplinary computational techniques that include molecular modeling and bioinformatics analysis. Molecular modeling makes it possible to create atomistic models of different biomolecular complexes, such as a DNA transcription factor protein complex, and observe how molecular interactions in these complexes develop over time under different conditions. We also develop new molecular modeling tools, which allow to recreate short-lived structures of intermediate stages and build detailed models of biological processes in a time- and resource-efficient way. Bioinformatics analysis makes it possible to collect information about which biomolecules interact with each other in the cell, and what will be the result of these interactions. Together, these methods create a synergistic effect and lead us to new exciting discoveries.

As a result of our research, we aim to gain new insights into how the regulation of the DNA transcription reaction works at the molecular mechanical level. We aim to develop new strategies and tools that can be useful for studies of other aspects of biological regulation, and for other researchers. And finally, we aim to develop new diagnostic tools that would allow time- and cost-efficiently to predict when a cell is under a risk to be transformed from healthy to pathogenic.

